user-info-provider/src/main/java/de/intension/mapper/user/UserInfoHelper.java

```diff
@@ -0,0 +1,446 @@
+package de.intension.mapper.user;
+
+import static de.intension.api.UserInfoAttribute.*;
+
+import java.nio.charset.StandardCharsets;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.jboss.logging.Logger;
+import org.keycloak.models.ProtocolMapperModel;
+import org.keycloak.models.UserModel;
+import org.keycloak.models.utils.KeycloakModelUtils;
+import org.keycloak.representations.IDToken;
+import org.keycloak.utils.StringUtil;
+
+import com.google.common.hash.Hashing;
+
+import de.intension.api.UserInfoAttribute;
+import de.intension.api.enumerations.*;
+import de.intension.api.json.*;
+
+public class UserInfoHelper
```

Comment:
woah big class - todo later

Reply:
let's keep it for now until we have a "stable"-release